### ISSUE TITLE
Steps for Hermes_Urls Registration

### ISSUE THREAD

**Issue by delscorcho** [jorgenpt/Hermes#10]
Are there documented steps for setting up the Hermes URI registration/association? For some, clicking the URI only takes them to the windows store. I didn't see any documentation on running the hermes_url register command, assuming it's needed for proper association. Thanks in advance!

**Comment by jorgenpt** [comment]
@delscorcho The URI registration / association should happen automatically the first time you start the editor with the plugin loaded -- no manual work required.

The registration itself happens in [`FWindowsHermesServerModule::RegisterScheme`](https://github.com/jorgenpt/Hermes/blob/main/HermesCore/Source/HermesServer/Private/Windows/WindowsHermesServer.cpp#LL185), where it calls `hermes_urls.exe register`. If you start the editor with `-LogCmds="LogHermesServer verbose"`, it should log that it does so you can verify.

The call to register comes from [`FGenericHermesServer::UpdateScheme`](https://github.com/jorgenpt/Hermes/blob/d29ccc4937ec14b67f4038af80c33e7c7368296e/HermesCore/Source/HermesServer/Private/GenericHermesServer.cpp#LL268), which should happen in `FGenericHermesServer::StartupModule`.


To help troubleshoot `hermes_urls.exe`, you can go Edit > Preferences in Unreal and find "Hermes URLs" under plugins, and check "Enable Debug Logging". If you restart the editor after enabling that, it'll instruct `hermes_urls.exe` to log -- I believe that goes into `Project/Plugins/HermesCore/Binaries/Win64/hermes.log`.

Let me know if that helped or you're still having intermittent luck getting it to work.

**Comment by delscorcho** [comment]
Yes, thank you, we did get to the bottom of it. The code you mentioned that handles registration was failing due to anti-malware software on some of our team's machines. We got that resolved. Thanks again for your time!
